repository/src/main/java/com/outdoor/foodcalc/domain/repository/plan/FoodPlanRepo.java

```diff
@@ -1,46 +1,118 @@
 package com.outdoor.foodcalc.domain.repository.plan;
 
 import com.outdoor.foodcalc.domain.model.plan.FoodPlan;
+import com.outdoor.foodcalc.domain.repository.AbstractRepository;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
 import org.springframework.stereotype.Repository;
 
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
 import java.time.ZonedDateTime;
 import java.util.List;
 import java.util.Optional;
 
 @Repository
-public class FoodPlanRepo implements IFoodPlanRepo {
+public class FoodPlanRepo extends AbstractRepository<FoodPlan> implements IFoodPlanRepo, RowMapper<FoodPlan> {
+
+    static final String SELECT_ALL_FOOD_PLANS_SQL = "select * from food_plan";
```

Comment:
I assume you have to specify all columns explicitly, like Products
select p.id as productId, p.name as productName,